### PR TITLE
Package ott.0.32

### PR DESCRIPTION
**Pull request by tperami**
# `ott.0.32`

Homepage: http://www.cl.cam.ac.uk/~pes20/ott/
Source repo: https://github.com/ott-lang/ott
Bugs: https://github.com/ott-lang/ott/issues
Changelog: https://github.com/ott-lang/ott/blob/master/revision_history.txt

### CODE REVIEW

File: packages/ott/ott.0.32/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "Thibaut Pérami <Thibaut.Perami@cl.cam.ac.uk>"
+authors: ["Peter Sewell" "Francesco Zappa Nardelli" "Scott Owens"]
+license: "part BSD3, part LGPL 2.1"
```
Comment (kit-ty-kate) [inline packages/ott/ott.0.32/opam:4]:
```suggestion
license: ["BSD-3-Clause" "LGPL-2.1-only"]
```

**Comment by tperami** [comment]
Just checking, but is having two licence in the licence field like `license: ["BSD-3-Clause" "LGPL-2.1-only"]` representative that some parts of the code are under one licence and other parts of the code are under the other licence? I was just afraid that it would implicitely mean that all the code can be available under any of those two licences

**Comment by kit-ty-kate** [comment]
It would be equivalent to `BSD-3-Clause AND LGPL-2.1-only` using the SPDX description format (see https://github.com/ocaml/opam/blob/9974973ece9c8fb20afa0cb5283328dc4bf92af1/doc/pages/Manual.md#opam for the the opam specific documentation)

**Comment by tperami** [comment]
I need to check with the authors if they are fine with that, don't merge now, please

**Comment by kit-ty-kate** [comment]
Thanks. Could you return the fix to the opam file upstream so it can be taken into account for next time?

**Comment by tperami** [comment]
Yes, I will do that, no problem

**Pull request merged**